config/prow/cluster/starter/starter-azure.yaml

```diff
@@ -1286,9 +1286,15 @@ spec:
         - --console-address=:"<<CHANGE_ME_MINIO_CONSOLE_PORT>>"
         env:
         - name: MINIO_ROOT_USER
-          value: "<<CHANGE_ME_AZURE_STORAGE_ACCOUNT_USER>>"
+          valueFrom:
+            secretKeyRef:
+              name: minio-root-creds
+              key: user
```

Comment:
`user` is specific to an installation (line 1292), please use `<<` and `>>` to indicate as such

Reply:
this is a key to read the value from secret. This should not be edited. Thats why its not enclosed with `<<>>`